types/heremaps/index.d.ts

```diff
@@ -5694,7 +5694,7 @@ declare namespace H {
              * @param opt_locale {(H.ui.i18n.Localization | string)=} - the language to use (or a full localization object).
              * @returns {H.ui.UI} - the UI instance configured with the default controls
              */
-            static createDefault(map: H.Map, mapTypes: H.service.Platform.MapTypes | H.service.MapType | H.service.DefaultLayers, opt_locale?: H.ui.i18n.Localization | string): H.ui.UI;
+            static createDefault(map: H.Map, mapTypes: H.service.Platform.MapTypes | H.service.DefaultLayers, opt_locale?: H.ui.i18n.Localization | string): H.ui.UI;
```

Comment:
I'm not sure whether or not the `mapTypes` parameter takes the extra marker and tile layers in the DefaultLayers interface, I'm pretty sure you just pass it an object of key-map-type pairs. If this is the case, it is probably better conceptually to separate the two interfaces and keep `mapTypes: H.service.Platform.MapTypes`.